app/services/hyrax/work_uploads_handler.rb

```diff
@@ -92,7 +92,6 @@ def attach
       acquire_lock_for(work.id) do
         event_payloads = files.each_with_object([]) { |file, arry| arry << make_file_set_and_ingest(file) }
         @persister.save(resource: work)
-        Hyrax.publisher.publish('object.metadata.updated', object: work, user: files.first.user)
```

Comment:
this one should stay!

it's necessary since we save immediately above it.

Reply:
oh right!